packages/common/src/utils/offers.ts

```diff
@@ -1,5 +1,12 @@
+import type { Offer, Order } from '../../types/checkout';
 import type { MediaOffer } from '../../types/media';
 
+import { formatPrice } from './formatting';
+
+export const getOfferPrice = (offer: Offer) => formatPrice(offer.customerPriceInclTax, offer.customerCurrency, offer.customerCountry);
+
+export const isSVODOffer = (offer: Offer | Order) => offer.offerId?.[0] === 'S';
```

Comment:
Although not wrong, this is Cleeng-specific. We probably want to create a custom offer type with a property like;

```ts
type Plan {
  // true when the plan is a subscription and billed each billing cycle
  isRecurring: boolean;
  // ...
}

Reply:
I only moved this code, but yes I agree. Another moment I think.